### DISCUSSION TITLE
Mirth Connect Administrator License Document

### DISCUSSION THREAD

**Discussion by jsabanal** [nextgenhealthcare/connect#5341] [Q&A] [answered]
Hi folks,

I just want to ask if there's any document that stating this software is Free of use and it doesn't need to have a license to use/install because it's asked by our compliance team. They just want to have a document to reference with so that we won't violate any legal concern when we use and install this software.

Maybe there's a link where can I find this which I cannot find but I much appreciate if anyone can point it to me where.

Thank you in advanced!

**Comment by jonbartels** [comment, marked as answer]
The MCAL installation directory has a `LICENSE` file at `java/app/docs/LICENSE`. It was likely shown to you at installation time for MCAL (unless you used the tarball). 

As your team reviews this, I would be curious to learn if the compliance team has any gripes about the license.

MCAL is a great supporting tool for MC. Java WebStart was getting to be a pain and inventing MCAL was a good boost. Alternatives exist but they aren't as good or as usable.

**Reply by siddharthsharma9258** [reply]
so .jnlp files have been replaced by MCAL?

**Reply by jonbartels** [reply]
MCAL manages connections and cache for JNLP files. Mirth still uses Webstart and JNLP but launching Mirth Connect from a JNLP with MCAL is a lot easier.

**Reply by jsabanal** [reply]
@jonbartels Thank you for your reply. Yes, I gave the `LICENSE` file and so far no feedback yet. I'm waiting for them to get back to me. For now, I'll keep this thread open for couple of days.

**Reply by tonygermano** [reply]
> Mirth still uses Webstart and JNLP but launching Mirth Connect from a JNLP with MCAL is a lot easier.

Slight correction, the Admin Launcher does still download and process the JNLP file, but it does not use Webstart to do it.

**Comment by jsabanal** [comment]
@jonbartels they saw the LICENSE file and question below: 

```
NEXTGEN HEALTHCARE hereby grants to you, and you accept, a
nonexclusive, personal license to use the Software, provided
in machine-readable, object-code form, only as authorized in
this Agreement. No source code will be provided and/or made
available to You.
```

They asking how it applies to commercial use?

**Reply by jonbartels** [reply]
I am not a lawyer and I cannot answer that.

I know hundreds of users that use MCAL at work and it's not a problem.

**Reply by jsabanal** [reply]
@jonbartels true. I agree on that. I do apologize for asking you a question something like that. I really do appreciate your response. 

Thank you.
